packages/base-map/package.json

```diff
@@ -9,7 +9,7 @@
   "module": "esm/index.js",
   "private": false,
   "dependencies": {
-    "@opentripplanner/core-utils": "^3.0.4",
+    "@opentripplanner/core-utils": "^4.0.0",
```

Comment:
Should be 4.1.0 now, sorry for the confusion.

Reply:
No worries. Version bumped in c109f32